docs/sources/flow/reference/stdlib/json_path.md

```diff
@@ -0,0 +1,36 @@
+---
+aliases:
+- ../../configuration-language/standard-library/json_path/
+title: json_path
+---
+
+# json_path
+
+The `json_path` function lookup values using [jsonpath](https://goessner.net/articles/JsonPath/) syntax.
+
+The function expects 2 strings. First string is the json string to lookup values from. Second string is the jsonpath expression.
+
+`json_path` always return a list of values. If the jsonpath expression does not match any values, an empty list is returned.
+
+> Remember to escape double quotes when passing JSON string literals to `json_path`.
```

Comment:
It's implied, but could we also mention that we _can_ pass the string export of another component directly if it's valid JSON?

Reply:
I do a bit copy/paste from json_encode. Is this enough?